### PR TITLE
Enable gcloud testing in docker

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: tests/test-sessions/test-gcloud-sessions/pom.xml [hunk 1]
```diff
@@ -78,6 +94,7 @@
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
               <skipTests>false</skipTests>
+              <trimStackTrace>false</trimStackTrace>
               <systemPropertyVariables>
                 <DATASTORE_DATASET>jetty9-work</DATASTORE_DATASET>
                 <DATASTORE_HOST>http://localhost:8088</DATASTORE_HOST>
```
Comment (janbartel) [inline tests/test-sessions/test-gcloud-sessions/pom.xml]: I don't think lines 99-100 are needed anymore are they?

File: tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanTestSupport.java [hunk 2]
```diff
@@ -25,8 +25,6 @@
 import java.util.Properties;
 
 import org.eclipse.jetty.toolchain.test.FS;
-import org.eclipse.jetty.toolchain.test.jupiter.WorkDir;
-import org.eclipse.jetty.toolchain.test.jupiter.WorkDirExtension;
 import org.eclipse.jetty.util.IO;
```
Comment (janbartel) [inline tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanTestSupport.java:28]: Not a fan of making changes to totally unrelated code ... but as they're unused imports I guess its ok.

**Review by janbartel — CHANGES_REQUESTED** [review]
Remove the unnecessary lines for the surefire plugin in the pom and its good.

File: tests/test-sessions/test-gcloud-sessions/pom.xml (hunk 1 above)
Comment (olamy) [reply tests/test-sessions/test-gcloud-sessions/pom.xml]: ah yup. We will need that in the top pom :)

File: tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanTestSupport.java (hunk 2 above)
Comment (joakime) [reply tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanTestSupport.java:28]: This looks like changes from a merge.

Comment (olamy) [reply tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanTestSupport.java:28]: @joakime nope I was just looking at this class and removed unused imports
